less/jumbotron.less

```diff
@@ -5,13 +5,17 @@
 
 .jumbotron {
   padding: @jumbotron-padding 0;
-  color: @jumbotron-color;
-  background-color: @jumbotron-bg;
   transition: background-color 0.2s ease-in;
 
-  h1,
-  .h1 {
-    color: @jumbotron-heading-color;
+  &.bg-primary, &.bg-info, .jumbotron-image & {
```

Comment:
cool, i'll remove it and test 👍 
edit: leaving jumbotron-image in there, that one's actually needed to "simulate" bg-primary